governance/polls/Ratification Poll for Adding Sidestream Auction Services Core Unit, SAS-001 - November 8, 2021.md

```diff
@@ -32,18 +32,27 @@ A brief summary of this proposal has been provided by the MIP Author and is show
 *The SAS-001 Core Unit aims to improve security, transparency and accessibility of the Maker Protocol by providing and maintaining auction services through open-source development.*
 
 Please review the links below to inform your position on this proposal before voting.
-* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP39/MIP39c2-Subproposals/MIP39c2-SP25.md)
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/1bde6b8c017ff216805f8a6aa542161d08259207/MIP39/MIP39c2-Subproposals/MIP39c2-SP25.md)
 * [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP25)
 * [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp25-adding-sidestream-auction-services-core-unit-sas-001/10796)
 
+### MIP40c3-SP43
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP43 adds the DAI budget for Core Unit SAS-001: Sidestream Auction Services.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP40/MIP40c3-Subproposals/MIP40c3-SP43.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP43)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp43-adding-sidestream-auction-services-core-unit-budget-sas-001/10799)
+
+
 ### MIP40c3-SP44
 
 A brief summary of this proposal has been provided by the MIP Author and is shown below:
 
-*MIP40c3-SP44 adds the MKR Incentive Plan budget for SAS-001: Sidestream Auction Services Core Unit. It contains:*
-*- Total MKR Expenditure Cap*
-*- Estimated MKR Expenditure (based on the current team)*
-*- Escrow Wallet mechanism*
+*MIP40c3-SP44 adds the MKR Incentive Plan budget for SAS-001: Sidestream Auction Services Core Unit. It contains: - Total MKR Expenditure Cap - Estimated MKR Expenditure (based on the current team)- Escrow Wallet mechanism.*
```

Comment:
*MIP40c3-SP44 adds the MKR Incentive Plan budget for SAS-001: Sidestream Auction Services Core Unit. It contains:*
- *Total MKR Expenditure Cap*
- *Estimated MKR Expenditure (based on the current team)*
- *Escrow Wallet mechanism*